static/site.js

```diff
@@ -1,4 +1,6 @@
 (function () {
+  const loginLink = document.getElementsByClassName('js-login-link')[0];
```

Comment:
Two things, 1) probably just use an ID instead of a class name 2) the code assumes the button will be there and calls loginLink.onclick which throws an error on the login pop up because it isn't there.